### PR TITLE
Adding CI builds for Python 3.10 in Github actions

### PR DESCRIPTION
**Pull request by holtgrewe**

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -81,6 +81,9 @@ Full Change List
 - Adding shortcut buttons to phenotype annotation (#289)
 - Fixing issue with multiple added variants (#283)
 - Implementing several usability improvements for clinvar submission editor (#286).
+- Implementing several usability improvements for clinvar submission editor (#286)
+- Make clinvar UI work with many annotations by making it load them lazily for one case at a time (#302)
```
[the remark this answers is not in the document]
Comment (holtgrewe) [reply HISTORY.rst]: good catch, fixed

**Comment by holtgrewe** [comment]
Had to bump pandas/numpy dependencies.

**Comment by holtgrewe** [comment]
So, numpy/pandas require Python >=3.8 these days...

**Review by stolpeo — APPROVED** [review]
LGTM (only can comment the stuff that was changed, not what should have been changed though).

**Comment by holtgrewe** [comment]
Failure is unrelated.

**Comment by holtgrewe** [comment]
Failure is unrelated.

**Pull request merged**